viewflow_extensions/management/commands/flow_graph.py

```diff
@@ -49,10 +41,15 @@ def handle(self, **options):
             raise CommandError("Could not find file %s" % (file_path, ))
         except (AttributeError, TypeError):
             raise CommandError("Could not find the flow with the name %s" % (flow_name, ))
-
-        flow_graph = FlowGraph(flow_cls)
-        graph = flow_graph.create_diagraph()
-        if options.get('svg'):
+        try:
+            from viewflow import chart
+        except ImportError:
+            from viewflow_extensions.flow_graph import FlowGraph
+            flow_graph = FlowGraph(flow_cls)
+            graph = flow_graph.create_diagraph()
             graph.render(filename='{}'.format(flow_path[0]))
         else:
-            self.stdout.write(text_type(graph))
+            grid = chart.calc_layout_data(flow_cls)
+            svg = chart.grid_to_svg(grid)
+            with open(flow_path[0], 'w') as f:
+                f.write("%s.svg" % svg)
```

Comment:
You add .svg to the file content instead of its name. File ends up corrupted and w/o extension. I think this is why the other test `test_create_graph_svg` fails.